### PR TITLE
Hotfix/docs Updates Terminal Intro for Options, Creates Terminal Intro for Mutual Funds

### PR DESCRIPTION
**Pull request by deeleeramone**
This PR updates the Terminal Intro Guide for Equity Options, and removes the guide for `Pricing` which no longer exists as a sub-menu or function.

Tagging on a new addition, Intro Guide for Mutual Funds menu.

### CODE REVIEW

File: website/content/terminal/usage/intros/stocks/options/index.md [hunk 1]
```diff
@@ -1,268 +1,154 @@
 ---
 title: Options
 keywords: [Options, stocks, derivatives, puts, calls, oi, vol, greeks, voi, volatility, vsurf, chains, parity, binom, screen, pricing, hedge, pcr, info, hist, grhist, plot, parity, how to, example, navigation]
-excerpt: This guide introduces the user to Options submenu, within the Stocks menu.
+excerpt: This guide introduces the Options sub-menu, within the Stocks menu, and provides examples for use.
 ---
-The Options menu provides the user with a comprehensive set of tools for analyzing equity options. This guide provides an overview of the menu and demonstrates commands in context. Using this menu correctly will require understanding terminology and math specific to the asset class. Wikipedia is a great resource for definitions and for learning about the mechanics of derivatives, read it <a href="https://en.wikipedia.org/wiki/Option_(finance)" target="_blank" rel="noreferrer noopener">here</a>. These are complex, leveraged, financial instruments requiring specialized knoweledge and a different frame-of-mind than the approach of an equities long-only investor. Always conduct thorough due diligence.<br />
+## Overview
+
```
Comment (jmaslek) [inline website/content/terminal/usage/intros/stocks/options/index.md:7]: knowledge spelling

File: website/content/terminal/usage/intros/stocks/options/index.md [hunk 2]
```diff
@@ -1,268 +1,154 @@
...
+```
+
+```console
+Getting Option Chain ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
+Loaded option chain from YahooFinance
+```
+
+#### pcr
+
+The `pcr` command plots a rolling put/call ratio (ten years max) over a selectable window of time (default is 30).
```
Comment (jmaslek) [inline website/content/terminal/usage/intros/stocks/options/index.md:106]: 30 days

File: website/content/terminal/usage/intros/stocks/options/index.md (hunk 1 above)
Comment (deeleeramone) [reply website/content/terminal/usage/intros/stocks/options/index.md:7]: corrected

File: website/content/terminal/usage/intros/stocks/options/index.md [hunk 3]
```diff
@@ -1,268 +1,154 @@
...
+### The Options Menu
+
+Navigate to the menu by typing `options`, from the `Stocks` menu, and then pressing enter. Alternatively, absolute path navigation can jump straight there, from anywhere. `/stocks/options`.  The source for the options data can be defined using the `load` command, or a default preference can be defined using the `/sources` menu.  The chains data returned will vary by source.  The following sources are currently available to use:
+
+- YahooFinance
+- Nasdaq
+- Tradier (requires API key)
+- Intrinio (requires API key)
+
+The menu can be entered without a ticker symbol loaded.  If one is already loaded from the stocks menu, it will automatically fetch the chains data from the default source.    The commands displayed above the currently active Ticker and Expiry do not require any data to be loaded, while the functions below do.
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/terminal/usage/intros/stocks/options/index.md:19]: added a note standout for this.

File: website/content/terminal/usage/intros/stocks/options/index.md [hunk 4]
```diff
@@ -1,268 +1,158 @@
...
+|:---------|:-----------|---------:|:-------|---------:|------:|-----:|------:|------:|
+| TFC      | 2023-05-19 |     34   | Call   |    127.2 | 15904 |  125 |  0.4  |  0.5  |
+| SCHW     | 2023-06-02 |     40   | Put    |     55.5 | 11996 |  216 |  0.28 |  0.34 |
+| TSLA     | 2023-05-05 |    149   | Put    |     49.3 | 57454 | 1165 |  2.93 |  2.95 |
+| TSLA     | 2023-04-28 |    157.5 | Call   |     46.8 | 99677 | 2132 |  1.39 |  1.4  |
+| USB      | 2023-05-19 |     32   | Call   |     44.9 |  6692 |  149 |  1.35 |  1.4  |
+| NOC      | 2023-05-19 |    440   | Put    |     43.8 |  4420 |  101 |  6    |  7.2  |
+| GM       | 2023-06-02 |     28   | Put    |     43.1 |  5867 |  136 |  0.32 |  0.35 |
+| AAPL     | 2023-05-19 |    167.5 | Call   |     41.6 | 10888 |  262 |  2.91 |  2.93 |
+| ATVI     | 2023-07-21 |     85   | Call   |     36.5 |  5772 |  158 |  1.59 |  1.71 |
```
Comment (jmaslek) [inline website/content/terminal/usage/intros/stocks/options/index.md:68]: Not sure I see the need for this table.  Takes a while to scroll through then you have another screenshot anyways.

Comment (deeleeramone) [reply website/content/terminal/usage/intros/stocks/options/index.md]: Removed the table.

**Review by jmaslek — APPROVED** [review]

**Pull request merged**